app/scripts/views/settings/emails.js

```diff
@@ -177,6 +163,12 @@ define(function (require, exports, module) {
 
   Cocktail.mixin(
     View,
+    UpgradeSessionMixin({
+      caption: t('A secondary email is an additional address for receiving security notices and confirming new Sync devices'),
+      gatedHref: 'settings/emails',
+      gatedTemplate: Template,
```

Comment:
This line can be removed now that you set `this.gatedTemplate = this.template` in the mixin.